docs_sphinx/advanced/custom_events.rst

```diff
@@ -27,23 +84,52 @@ model::
 
     event_mon = EventMonitor(group, 'custom_event', variables['var1', 'var2'])
 
+Triggering `NeuronGroup` code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 If the event should trigger a series of statements (i.e. the equivalent of
 ``reset`` statements), this can be added by calling `~NeuronGroup.run_on_event`::
 
     group.run_on_event('custom_event', 'x=0')
 
+Triggering synaptic pathways
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 When neurons are connected by `Synapses`, the ``pre`` and ``post`` pathways
-are triggered by spike events by default. It is possible to change this by
+are triggered by ``spike`` events on the presynaptic and postsynaptic `NeuronGroup`
+by default. It is possible to change which pathway is triggered by which event by
 providing an ``on_event`` keyword that either specifies which event to use for all
 pathways, or a specific event for each pathway (where non-specified pathways use
 the default ``spike`` event)::
 
     synapse_1 = Synapses(group, another_group, '...', on_pre='...', on_event='custom_event')
+
+The code above causes all pathways to be triggered by an event named ``custom_event``
+instead of the default ``spike``.
+
+::
+
     synapse_2 = Synapses(group, another_group, '...', on_pre='...', on_post='...',
                          on_event={'pre': 'custom_event'})
 
+In the code above, only the ``pre`` pathway is triggered by the ``custom_event``
+event.
+
+We can also create new pathways and have them be triggered by custom events.
+For example::
+
+    synapse_3 = Synapses(group, another_group, '...',
+                         on_pre={'pre': '....',
+                                 'custom_pathway': '...'},
+                         on_event={'pre': 'spike',
+                                   'custom_pathway': 'custom_event'})
+
+In this code, the default ``pre`` pathway is still triggered by the ``spike``
+event, but there is a new pathway called ``custom_pathway`` that is triggered
+by the ``custom_event`` event.
+
 Scheduling
-----------
+~~~~~~~~~~
```

Comment:
And again something that's not your fault but would be good to fix. The reference at the end uses ``:doc:`scheduling` `` but should use `` :ref:`scheduling` `` instead (otherwise it refers to "custom progress reporting"...).